hwi/aladin_crawling.py

```
@@ -0,0 +1,99 @@
+
+# ### 2. HTML 페이지 불러오기 & 파싱
+
+# In[109]:
+
+
+from bs4 import BeautifulSoup
+import requests
+
+# 알라딘 베스트셀러
+url = "https://www.aladin.co.kr/shop/common/wbest.aspx?BestType=Bestseller&BranchType=1&CID=0&page=1&cnt=1000&SortOrder=1"
+response = requests.get(url)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)